java-logging-spring/src/test/java/uk/gov/hmcts/reform/logging/spring/RequestLoggingComponentTest.java

```diff
@@ -50,24 +52,41 @@ public void listenForEvents() throws JoranException, IOException {
     @Test
     public void requestProcessedMessageShouldBeLoggedForPublicResource() throws Exception {
         ResponseEntity<String> response = restTemplate.getForEntity("/public", String.class);
+
         assertThat(response.getBody()).isEqualTo("OK");
-        assertThat(loggedEvents()).extracting("message").containsOnlyOnce("Request processed");
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
```

Comment:
This seems unnecessary as the test is about logging a request regardless of the response's status.